hack/local-development/local-garden/start.sh

```diff
@@ -46,4 +48,4 @@ $(dirname $0)/apply-rbac-garden-ns
 
 echo "# Now you can start using the cluster at with \`export KUBECONFIG=hack/local-development/local-garden/kubeconfigs/default-admin.conf\`"
 echo "# Then you need to run \`make dev-setup\` to setup config and certificates files for gardener's components and to register the gardener-apiserver."
-echo "# Finally, run \`make start-apiserver,start-controller-manager,start-scheduler,start-gardenlet\` to start the gardener components as usual."
\ No newline at end of file
+echo "# Finally, run \`make start-apiserver,start-controller-manager,start-scheduler,start-gardenlet\` to start the gardener components as usual."
```

Comment:
How about the following or something similar?
```
if $ACTIVATE_SEEDAUTHORIZER; then
  echo '# Additionally, make sure to run make start-admission-controller'
fi
```

Reply:
Will do